.travis.yml

```diff
@@ -86,5 +88,14 @@ jobs:
       install: doit package_build $CHANS_DEV $PKG_TEST_PYTHON --test-group=all
       script: doit package_upload --token=$CONDA_UPLOAD_TOKEN --label=dev
 
+    # release packages
+
+    - <<: *conda_default
+      stage: conda_package
+      env: DESC="" TRAVIS_NOCACHE=$TRAVIS_JOB_ID
+      install: doit package_build $CHANS_DEV $PKG_TEST_PYTHON --test-group=all
```

Comment:
I think the idea is to use chans_rel here (i.e. basically pyviz instead of pyviz/label/dev)